app/scripts/controllers/error.ctrl.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+angular.module('yourStlCourts').controller('ErrorCtrl', function (Error) {
+  var ctrl = this;
```

Comment:
You won't need to fetch the error from the Error service (again, don't store that error state - what if there is more than 1 error!?). Instead, when you call ```$state.go``` in the error handler, provide the error object to the state itself....i.e. ```$state.go('Error', {error: errorObject});``` and then similar to what you've used in the past, ```error``` will be on ```$stateParams```.